src/layers/PolygonSeries.js

```diff
@@ -361,6 +365,9 @@ export class PolygonSeries extends PaletteMixin(CoverageMixin(L.Layer)) {
    */
   redraw () {
     this._updatePolygon()
-    this._geojson.redraw()
+    
+    this._geojson.setStyle({

```

Comment:
`_updatePolygon()` should already update the style, currently by re-creating the polygon, so not sure what the additional `setStyle` would do here.

Reply:
Sorry, my bad, I added that while trying to figure out what was going wrong.  I'll remove it.